src/test/java/it/pagopa/ecommerce/commons/client/NpgClientTests.java

```diff
@@ -123,6 +123,181 @@ void shouldRetrieveFieldsDtoUsingExplicitParameters() {
                 .verifyComplete();
     }
 
+    @Test
+    void shouldRetrieveFieldsDtoUsingExplicitParametersWithLangExplicitSetIta() {
```

Comment:
Here you could use [`@ParameterizedTest`](https://www.baeldung.com/parameterized-tests-junit-5)

Reply:
Also: is there a test with `null` as the language parameter?

Reply:
yes, most tests here do not expect language as this was added only in this PR

Reply:
ParametrizedTest require JUnit 5 with is not currently included in the project.
We can take note of this and maybe open another PR specifically to improve the tests?

Reply:
I think we include JUnit 5 transitively, because we do already have some [parameterized tests](https://github.com/pagopa/pagopa-ecommerce-commons/blob/main/src/test/java/it/pagopa/ecommerce/commons/domain/BINTest.java#L24) 🤣
I would add them and include a new task in the backlog that we need to check versions' dependencies

Reply:
Right. i was looking in the POM and got misleaded. I added them now. Thanks